stm32_base/hal/ADC_stm32/ADC_device_stm32.c

```diff
@@ -22,15 +22,15 @@
 #define ADC_PORT_VOLTAGE_RANGE 		5.0f
 #define ADC_SYSTEM_VOLTAGE_RANGE	20.0f
 
-#define SCALING_5V 					0.00125691302f
-#define SCALING_BATTERYV			0.00465f
+#define SCALING_5V 		0.00125691302f
```

Comment:
Even alignment in blocks of definitions.  No hard and fast rule, just that a block of definitions with varying length have aligned constants.